wallet/src/de/schildbach/wallet/ui/dashpay/CreateIdentityService.kt

```diff
@@ -656,6 +656,7 @@ class CreateIdentityService : LifecycleService() {
         blockchainIdentityData = BlockchainIdentityData(CreationState.NONE, null, null, null, true)
 
         val authExtension = walletApplication.wallet!!.getKeyChainExtension(AuthenticationGroupExtension.EXTENSION_ID) as AuthenticationGroupExtension
+        //authExtension.setWallet(walletApplication.wallet!!) // why is the wallet not set?  we didn't deserialize it probably!
```

Comment:
This line is in danger of being abandoned here forever. Maybe `TODO` should be added?

Reply:
Thanks.  This line will be removed as the bug was fixed elsewhere.